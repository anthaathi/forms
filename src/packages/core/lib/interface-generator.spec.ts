import interfaceGenerator from './interface-generator';

describe('Interface generator', () => {
  it('should render imports', () => {
    const result = interfaceGenerator({
      'baseui/input': ['Input'],
      'baseui/layout-grid': ['Grid', 'Cell'],
    });

    expect(result).toMatchSnapshot();
  });
});
