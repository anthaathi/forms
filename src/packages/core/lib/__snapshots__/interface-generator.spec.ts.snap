// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Interface generator should render imports 1`] = `
"/* eslint-disable */
import * as React from \\"react\\";
import { Input } from \\"baseui/input\\";
import { Grid, Cell } from \\"baseui/layout-grid\\";
export type Input$Props = React.ComponentProps<typeof Input>;
export { Input };
export type Grid$Props = React.ComponentProps<typeof Grid>;
export { Grid };
export type Cell$Props = React.ComponentProps<typeof Cell>;
export { Cell };
export default [
 { type: \\"Input\\", component: Input  } , { type: \\"Grid\\", component: Grid  } , { type: \\"Cell\\", component: Cell  } 
]"
`;
